### PR TITLE
Update new tab section: rename Office 365 to Microsoft 365

### PR DESCRIPTION
**Pull request by vwehren**
Privacy white paper update for the New tab section: rename "Office 365" to "Microsoft 365" in line with current product language.

AB#45091642

### CODE REVIEW

File: microsoft-edge/privacy-whitepaper/index.md [hunk 1]
```diff
@@ -420,9 +420,9 @@ Microsoft Edge collects diagnostic data about how you use the new tab page, such
...
 
-#### Office 365
+#### Microsoft 365
 
-If you are signed in to Microsoft Edge with a work or school account, your organization may turn on Office 365 as an option for page content on the new tab page. The feature is currently available only for commercial customers, under the [Microsoft Online Services Terms (OST)](https://www.microsoft.com/licensing/product-licensing/products). For more information about privacy for Office 365, see [Overview of privacy controls for Microsoft 365 Apps for enterprise](/deployoffice/privacy/overview-privacy-controls).
+If you are signed in to Microsoft Edge with a work or school account, your organization may turn on Microsoft 365 as an option for page content on the new tab page. The feature is currently available only for commercial customers under the [Microsoft Online Services Terms (OST)](https://www.microsoft.com/licensing/product-licensing/products). For more information about privacy for Office 365, see [Overview of privacy controls for Microsoft 365 Apps for enterprise](/deployoffice/privacy/overview-privacy-controls).
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:425]:
```suggestion
If you are signed in to Microsoft Edge with a work or school account, your organization may turn on Microsoft 365 as an option for page content on the new tab page. The feature is currently available only for commercial customers under the [Microsoft Online Services Terms (OST)](https://www.microsoft.com/licensing/product-licensing/products). For more information about privacy for Microsoft 365, see [Overview of privacy controls for Microsoft 365 Apps for enterprise](/deployoffice/privacy/overview-privacy-controls).
```

File: microsoft-edge/privacy-whitepaper/index.md [hunk 2]
```diff
@@ -420,9 +420,9 @@ Microsoft Edge collects diagnostic data about how you use the new tab page, such
...
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:426]: change from "Office 365"?
https://learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/#new-tab-page

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/privacy-whitepaper/index.md [hunk 3]
```diff
@@ -420,9 +420,9 @@ Microsoft Edge collects diagnostic data about how you use the new tab page, such
...
 
-#### Office 365
+#### Microsoft 365
 
-If you are signed in to Microsoft Edge with a work or school account, your organization may turn on Office 365 as an option for page content on the new tab page. The feature is currently available only for commercial customers, under the [Microsoft Online Services Terms (OST)](https://www.microsoft.com/licensing/product-licensing/products). For more information about privacy for Office 365, see [Overview of privacy controls for Microsoft 365 Apps for enterprise](/deployoffice/privacy/overview-privacy-controls).
+If you are signed in to Microsoft Edge with a work or school account, your organization may turn on Microsoft 365 as an option for page content on the new tab page. The feature is currently available only for commercial customers under the [Microsoft Online Services Terms (OST)](https://www.microsoft.com/licensing/product-licensing/products). For more information about privacy for Microsoft 365, see [Overview of privacy controls for Microsoft 365 Apps for enterprise](/deployoffice/privacy/overview-privacy-controls).
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:426]: so far, the PR fixes the 3 instances of "Office 365" that are in section 
https://learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/#office-365
but hasn't yet fixed the instance in 
https://learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/#new-tab-page

**Review by mikehoffms — CHANGES_REQUESTED** [review]
added one more instance to fix.  also experimenting with the "Undo Approval" PR UI.

**Review by apryzgoda — APPROVED** [review]
Edge Privacy Approved

File: microsoft-edge/privacy-whitepaper/index.md [hunk 4]
```diff
@@ -420,9 +420,9 @@ Microsoft Edge collects diagnostic data about how you use the new tab page, such
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:420]: @vwehren - fix the instance of "Office 365" in the "New tab page" parent section above (line 403).
Live article section:
https://learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/#new-tab-page

File: microsoft-edge/privacy-whitepaper/index.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:426]: entered fresh [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2657/files#r1234562726) above the changed section

**Review by mikehoffms — APPROVED** [review]

**Review by captainbrosset — APPROVED** [review]

**Pull request merged**